core/interfaces/i_movable.ts

```diff
@@ -4,6 +4,8 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+import {Coordinate} from '../utils';
```

Comment:
Import from `'../utils/coordinate.js'`

Reply:
Thanks for the comment. I have changed accordingly and pushed it.